torch/ao/quantization/fx/utils.py

```diff
@@ -640,3 +641,220 @@ def get_skipped_module_name_and_classes(
         skipped_module_classes += get_custom_module_class_keys(prepare_custom_config.float_to_observed_mapping)
 
     return skipped_module_names, skipped_module_classes
+
+def _is_custom_module_lstm(node: Node, named_modules: Dict[str, torch.nn.Module]) -> bool:
+    """
+    Return whether this refers to the custom module LSTM flow.
+    """
+    if node.op != "call_module" or str(node.target) not in named_modules:
+        return False
+    mod = named_modules[str(node.target)]
+    return isinstance(mod, torch.nn.LSTM) or isinstance(mod, torch.ao.nn.quantizable.LSTM)
+
+def _is_getitem_node(node: Node) -> bool:
+    """
+    Return whether `node` refers to a `getitem` call_function node.
+    """
+    return node.op == "call_function" and node.target == operator.getitem
+
+def _is_dequant_stub(node: Node, named_modules: Dict[str, torch.nn.Module]) -> bool:
+    """
+    Return whether `node` refers to a `DeQuantStub` call_module node.
+    """
+    if node.op != "call_module" or str(node.target) not in named_modules:
+        return False
+    return isinstance(named_modules[str(node.target)], DeQuantStub)
```

Comment:
nit: I think the isinstance check should be inlined.  Otherwise people have to create a new util function for every type they want to check against.

Something like

```
from torch.ao.quantization.utils import getattr_from_fqn

mod = getattr_from_fqn(model, node.target)
if isinstance(mod, DeQuantStub):
    ...
```

would be cleaner